umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/dif10.clj

```diff
@@ -13,12 +13,18 @@
     [cmr.umm-spec.umm-to-xml-mappings.dif10.spatial :as spatial]
     [cmr.umm-spec.util :as u]))
 
+;; TODO?
 (def coll-progress-mapping
   "Mapping from known collection progress values to values supported for DIF10 Dataset_Progress."
   {"COMPLETE" "COMPLETE"
    "ACTIVE" "IN WORK"
    "PLANNED" "PLANNED"
-   "DEPRECATED" "COMPLETE"})
+   "DEPRECATED" "COMPLETE"
+   "NOT PROVIDED" "NOT PROVIDED"
+   "PREPRINT" "PREPRINT"
+   "INREVIEW" "INREVIEW"
+   "SUPERSEDED" "SUPERSEDED"
```

Comment:
These values need to be added to the DIF-10 schema.  The enumeration is called DatasetProgressEnum.